recipes/pp-sketchlib/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set name = "pp-sketchlib" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/johnlees/{{ name|lower }}/archive/v{{ version }}.tar.gz
+  sha256: 8032ad7b0ae591cfe44d469b415da998430894d780849ee09f01c3230084bfe4
+  patches:
+    - 0001-CMake.patch
+    - 0002-Install.patch  # [linux64 and cuda_compiler_version != "None"]
+    - 0003-Add-C-macros-to-Makefile.patch
+
+build:
+  number: 0
+  skip: True  # [py2k or win]
+  script:
+    - export CFLAGS="${CFLAGS} -I/usr/include"  # [linux64 and cuda_compiler_version != "None"]
```

Comment:
Why add `/usr/include`. We disallow using system headers.

Reply:
This was based on https://github.com/conda-forge/cupy-feedstock/blob/master/recipe/meta.yaml#L30. I can try removing it, but haven't quite managed to get it to the `nvcc` compile step yet

Reply:
Removed these lines now